var/spack/repos/builtin/packages/nwchem/package.py

```diff
@@ -30,12 +30,14 @@ class Nwchem(Package):
 
     variant("openmp", default=False, description="Enables OpenMP support")
     variant("mpipr", default=False, description="Enables ARMCI with progress rank")
+    variant("fftw3", default=False, description="Link against the FFTW library")
 
     # This patch is for the modification of the build system (e.g. compiler flags) and
     # Fortran syntax to enable the compilation with Fujitsu compilers. The modification
     # will be merged to the next release of NWChem (see https://github.com/nwchemgit/nwchem/issues/347
     # for more detail.
     patch("fj.patch", when="@7.0.2 %fj")
+    patch("fftw_splans.patch")
```

Comment:
Can you add a brief comment above the `patch` directive explaining:
1. What is the effect of the patch
2. Why it is needed
3. Where it was taken from

?

Reply:
* added patch documentation
* patch applies only to version=7.2.0